### ISSUE TITLE
[selectors] ::shadow no longer exists?

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#7474]
https://drafts.csswg.org/selectors-4/#pseudo-element-structure references a `::shadow` pseudo-element, defined in https://www.w3.org/TR/css-scoping-1/#selectordef-shadow

But that's an old working draft from 2014. I don't see `::shadow` in the editor's draft https://drafts.csswg.org/css-scoping-1/

So this section may need to be changed.

**Comment by fantasai** [comment]
@Loirooriol Fixed by switching to `::slotted`. Incidentally, the next example in the section... do you think there's something reasonable we could switch it to, or should I just comment it out?

**Comment by Loirooriol** [comment]
So, `::slotted()` is an alias for other elements in the tree, and then I guess it makes sense to have internal structure.

But I don't see https://drafts.csswg.org/css-scoping/#slotted-pseudo saying that it has internal structure.

Also, example 8 is still saying mentioning `::shadow`:

> the selectors `div > span` and `div::shadow ::slotted > span`

Also, AFAIK `::slotted` is not valid, it should be `::slotted(*)` or something.

**Comment by fantasai** [comment]
@Loirooriol Yeah, I realized that ::slotted doesn't currently have internal structure... that's under discussion somewhere I think. So I switched it to be a hypothetical pseudo-element that doesn't exist, since we don't have any examples yet.

But still, that second example, which relies on both ::slotted and ::shadow existing, do I just comment it out or is there some valid useful thing you think we could replace it with?
